src/web-stack/apps/positive-affirmations/src/app/users/entities/affirmation.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { UserEntity } from './user.entity'
+
+@Entity({ name: 'user' })
+export class AffirmationEntity {
+    @PrimaryGeneratedColumn('uuid')
+    public id: string;
+
+    // Reference for column types to database compatibility:
+    // https://github.com/typeorm/typeorm/blob/master/src/driver/types/ColumnTypes.ts
+    @Column('text')
+    public title: string;
+
+    @Column('text', {nullable: true})
+    public note: string;
+
+    @Column('int')
+    public likes: number;
+
+    // TODO how to do this?
+    // @Column({name: 'created_by', type: 'user'})
+    // public createdBy: string;
+    
+    /**
+     * TODO These are computed values and not just numbers. Link to reaffirations
+     */
+    @Column('int')
+    public reaffirmations: number;
+
+    /**
+     * TODO Not entirely sure about this. Verify!
+     */
+    @Column({name: 'picture_filename', type: 'text', nullable: true})
+    public profilePicturePath: string;
```

Comment:
I recommend we define a new entity for bucket files and the profile picture type defined here could be a bucket file.

You can check here for entity definition I did for bucket files on a previous project. Its in C#, but the types are clear enough:

```
public class BucketFile : AuditableEntity
    {
        public string BucketFileId { get; set; }
        public string BucketName { get; set; }
        public string OriginalFileName { get; set; }
        public string FilePath { get; set; }
        public bool Uploaded { get; set; }
        // TODO: Create enums using standard types for the below metadata. See here https://cloud.google.com/storage/docs/metadata?_ga=2.147137079.-1682943862.1576792853&_gac=1.238357172.1589199078.EAIaIQobChMI18L25OOr6QIVmYKyCh2YbwP7EAAYASABEgIU_PD_BwE
        public string ContentTypeMetadata { get; set; }
        public string ContentEncodingMetadata { get; set; }
        public string ContentLanguageMetadata { get; set; }
        public string CacheControlMetadata { get; set; }

        public int SizeInBytes { get; set; }

        public string UiTitle { get; set; }
        public string UiDescription { get; set; }

        public bool Archived { get; set; }
    }
```

```
    public class AuditableEntity
    {
        public string CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public string LastModifiedBy { get; set; }
        public DateTime? LastModifiedOn { get; set; }
    }
```

Reply:
https://cloud.google.com/storage/docs/creating-buckets